src/evaluatorHelpers.ts

```diff
@@ -276,17 +276,45 @@ export async function renderPrompt(
   } else if (prompt.raw.startsWith('langfuse://')) {
     const langfusePrompt = prompt.raw.slice('langfuse://'.length);
 
-    // we default to "text" type.
-    const [helper, version, promptType = 'text'] = langfusePrompt.split(':');
+    let helper: string;
+    let version: string | undefined;
+    let label: string | undefined;
+    let promptType: 'text' | 'chat' | undefined = 'text';
+
+    // More robust parsing that handles @ in prompt IDs
+    // Look for the last @ that's followed by a label pattern
+    const labelMatch = langfusePrompt.match(/^(.+)@([^:@]+)(?::(.+))?$/);
+    const versionMatch = langfusePrompt.match(/^([^:]+):(\d+|latest)(?::(.+))?$/);
+
+    if (labelMatch) {
+      // Label-based syntax: prompt-id@label or prompt-id@label:type
+      helper = labelMatch[1];
+      label = labelMatch[2];
+      if (labelMatch[3]) {
+        promptType = labelMatch[3] as 'text' | 'chat';
+      }
+    } else if (versionMatch) {
+      // Version-based syntax: prompt-id:version or prompt-id:version:type
+      helper = versionMatch[1];
+      version = versionMatch[2];
+      if (versionMatch[3]) {
+        promptType = versionMatch[3] as 'text' | 'chat';
+      }
+    } else {
+      // Simple prompt-id only
+      helper = langfusePrompt;
+    }
+
```

Comment:
nit: an alternative way to achieve this would have been to leverage the fact that labels are always [strings](https://api.reference.langfuse.com/#tag/prompts/get/api/public/v2/prompts/{promptName}) and versions are always [integers](https://api.reference.langfuse.com/#tag/prompts/get/api/public/v2/prompts/{promptName}) and you can only specify one of them. This would then require minimal changes as the current format `langfuse://my-prompt:<integer_version_or_label_string>:(text|chat)` would work with both.

Reply:
But regardless, thank you implementing this so quickly!